README.md

```diff
@@ -14,15 +14,69 @@ The goal of this project is to develop an efficient data pipeline that streamlin
 - **Growth Rate of Commits:** This metric measures the speed at which new commits are being made to a repository.
 - **Custom Analysis:** This allows for the implementation of custom metrics or analysis tailored to specific project requirements.
 
-
 ### Source Data
 
-The project utilizes data from the GH Archive, a third-party source, due to its advantages of simple load logic compared to GitHub's native data. Despite being a third-party source, it offers timely updates, refreshing every hour. We update our data daily to ensure the latest information is utilized. We mitigate the risk associated with third-party data sources by implementing rigorous data quality tests.
+The project utilizes data from the GH Archive, a third-party source, due to its advantages of simple load logic compared to GitHub's native data. GH Archive offers timely updates, refreshing every hour, and we update our data daily to ensure the latest information is utilized. To mitigate risks associated with third-party data sources, we implement rigorous data quality tests.
 
 ### Data Format
 
 The data is delivered in the form of SQL databases.
 
 ### Update Frequency
 
-The data is updated daily to maintain up-to-date information for analysis.
\ No newline at end of file
+The data is updated daily to maintain up-to-date information for analysis.
+
+## Model
+
+In this project, we will be utilizing the **Kimball Model**, aka "Star Schema."
+
+### Kimball Model Components
+
+- **Grain**: The grain establishes exactly what a single fact table row represents. It defines the level of detail captured in the data.
+  
+- **Dimensions**: Dimensions provide the “who, what, where, when, why, and how” context surrounding a business process event. They represent the various attributes that describe the event.
+
+- **Facts**: Facts are the measurements that result from a business process event and are almost always numeric. They quantify the business activities recorded in the data warehouse.
+
+### Facts and Dimension Tables
+
+#### Facts: 
+
+- **Stars:**
```

Comment:
Naming: Let's try to use `fact_` and `dim_` prefixes for our tables, so that end users who are familiar with Kimball can quickly find their way around.